### ISSUE TITLE
Netty tcnative build for Mac fails

### ISSUE THREAD

**Issue by dineshappavoo** [netty/netty-tcnative#301]
I followed the http://netty.io/wiki/forked-tomcat-native.html#building-on-mac-os-x wiki to build netty-tcnative with a static openssl. I got some error with the build,

```
[Dinesh netty-tcnative]$ xcode-select --install
xcode-select: error: command line tools are already installed, use "Software Update" to install updates

[Dinesh netty-tcnative]$ brew install autoconf automake libtool openssl perl ninja golang cmake apr
Warning: autoconf-2.69 already installed
Warning: automake-1.15 already installed
Warning: libtool-2.4.6 already installed
Warning: openssl-1.0.2e_1 already installed
Warning: perl-5.22.1 already installed
Warning: ninja-1.6.0 already installed
Warning: go-1.5.3 already installed
Warning: cmake-3.4.1 already installed
Warning: apr-1.5.2 already installed

[Dinesh netty-tcnative]$ git branch
* master

[Dinesh netty-tcnative]$ mvn clean -e install
[INFO] Error stacktraces are turned on.
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Detecting the operating system and CPU architecture
[INFO] ------------------------------------------------------------------------
[INFO] os.detected.name: osx
[INFO] os.detected.arch: x86_64
[INFO] os.detected.classifier: osx-x86_64
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO]
[INFO] Netty/TomcatNative [Parent]
[INFO] Netty/TomcatNative [OpenSSL - Dynamic]
[INFO] Netty/TomcatNative [OpenSSL - Static]
[INFO] Netty/TomcatNative [BoringSSL - Static]
[INFO] Netty/TomcatNative [LibreSSL - Static]
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Building Netty/TomcatNative [Parent] 2.0.7.Final-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO]
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ netty-tcnative-parent ---
...
...
...
...
...
...
...
...
...
...
...
...
[INFO] Finished at: 2017-08-25T11:48:11-07:00
[INFO] Final Memory: 20M/301M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.fusesource.hawtjni:maven-hawtjni-plugin:1.11:build (build-native-lib) on project netty-tcnative: build failed: org.apache.maven.plugin.MojoExecutionException: ./configure failed with exit code: 1 -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.fusesource.hawtjni:maven-hawtjni-plugin:1.11:build (build-native-lib) on project netty-tcnative: build failed: org.apache.maven.plugin.MojoExecutionException: ./configure failed with exit code: 1
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:212)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.apache.maven.plugin.MojoExecutionException: build failed: org.apache.maven.plugin.MojoExecutionException: ./configure failed with exit code: 1
	at org.fusesource.hawtjni.maven.BuildMojo.execute(BuildMojo.java:254)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:207)
	... 20 more
Caused by: org.apache.maven.plugin.MojoExecutionException: ./configure failed with exit code: 1
	at org.fusesource.hawtjni.maven.BuildMojo.configureBasedBuild(BuildMojo.java:370)
	at org.fusesource.hawtjni.maven.BuildMojo.execute(BuildMojo.java:245)
	... 22 more
[ERROR]
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :netty-tcnative
```

Could you help me with this issue?

**Comment by Scottmitch** [comment]
can you run `./configure` manually, or is the output captured in a log file ... if would be helpful to know why configure failed.

**Comment by dineshappavoo** [comment]
@Scottmitch  Thanks for looking into this issue. It looks like I was in the wrong directory to run the `mvn clean install`. I ran from the top level netty-tcnative folder. Then I moved to `netty-tcnative/openssl-static` folder and ran mvn clean install, it worked. 

Also it would be helpful add a note in the wiki to switch to the folder {openssl-static, openssl-dynamic etc} before running `mvn clean install`
